gemfiles/rack_1.5.2.gemfile

```diff
@@ -1,33 +1,33 @@
 # This file was generated by Appraisal
 
-source 'https://rubygems.org'
+source "https://rubygems.org"
 
-gem 'rack', '1.5.2'
+gem "rack", "1.5.2"
```

Comment:
Adding `gemfiles/*` to `rubocop.yml` auto correct the `"` problem, however rubocop will compain about the syntax of gemfiles.